README.md

```diff
@@ -26,3 +28,7 @@ page](https://my.sailthru.com/settings/universal_links).
 When making changes, you should do so in the `apple-app-site-association.json`
 file and then run `yarn build` to encode new paths for Sailthru or remove old
 ones.
+
+## Caching
+
+Apple caches the apple-app-site-association file on app install. This means any updates to the `apple-app-site-association` file will only affect users who have recently installed the app, updated the app, or turned Universal Linking [off (and on again)](https://stackoverflow.com/questions/32729489/how-can-i-reset-ios-9-universal-linking-settings). More details about how Apple caches this file can be found on this [Stack Overflow answer](https://stackoverflow.com/a/41305871).
```

Comment:
It definitely updates periodically in between installs/updates, I _think_ it’s daily but in true Apple fashion it’s not documented. The main issue remains, of course, it’s not necessarily updated _when we would like it to_.